sqlx-postgres/src/column.rs

```diff
@@ -10,11 +10,21 @@ pub struct PgColumn {
     pub(crate) name: UStr,
     pub(crate) type_info: PgTypeInfo,
     #[cfg_attr(feature = "offline", serde(skip))]
-    pub(crate) relation_id: Option<i32>,
+    pub(crate) relation_id: Option<crate::types::Oid>,
     #[cfg_attr(feature = "offline", serde(skip))]
     pub(crate) relation_attribute_no: Option<i16>,
 }
 
+impl PgColumn {
+    pub fn relation_id(&self) -> Option<u32> {
```

Comment:
`Oid` is a public type, you can just return it directly: https://docs.rs/sqlx/latest/sqlx/postgres/types/struct.Oid.html

Reply:
Makes sense!